### PR TITLE
fixed major timing issue

### PR DESCRIPTION
**Pull request by joshzarrabi**
Stopped converting cursor to list and cut out a large time burden
This is much faster.
According to my time measurements the geojson conversion is still slow, I'll look into that now but its hard to tell on my data set, this might be enough

### CODE REVIEW

**Comment by shankari** [comment]
Testing?
In particular, I don't think that you can use the cursor directly like that, although I could be mistaken.

**Comment by joshzarrabi** [comment]
It works

**Comment by shankari** [comment]
I checked in big new datasets into the `data-collection-eval` repo. Use git lfs to check them out.
https://git-lfs.github.com/

**Comment by shankari** [comment]
I'd like some more detail than "it works".
what did you do to test it? what is the impact on the time?
some copy-pasted traces would be good.

**Comment by joshzarrabi** [comment]
74910 2016-04-04 14:35:31,144:DEBUG:START GET /result/heatmap/pop.route/commute/walking                                                                              
74911 2016-04-04 14:35:31,144:DEBUG:Filtering values for range 1325404800000 -> 1459666800000
74912 2016-04-04 14:35:31,146:INFO:About to query for 2012-01-01 00:00:00 -> 2016-04-03 00:00:00
74913 2016-04-04 14:35:31,438:DEBUG:result cursor has 10275 entries
74914 2016-04-04 14:35:31,496:DEBUG:Converted datetime range 2012-01-01 00:00:00 -> 2016-04-03 00:00:00 to timestamp range 1448640163.72 -> 1449037798.53
74915 2016-04-04 14:35:31,595:DEBUG:len(places) = 107, len(trips) = 107

**Comment by shankari** [comment]
This is a fix for https://github.com/e-mission/e-mission-server/issues/256

File: emission/storage/decorations/timeline.py [hunk 1]
```diff
@@ -61,12 +63,12 @@ def get_aggregate_timeline_from_dt(start_dt, end_dt, box=None):
         logging.info("About to query for %s -> %s in %s" % (start_dt, end_dt, box))
     result_cursor = edb.get_timeseries_db().find({"data.local_dt": {"$gte": start_dt, "$lte": end_dt}}).sort("metadata.write_ts")
     logging.debug("result cursor has %d entries" % result_cursor.count())
-    result_list = list(result_cursor)
-    logging.debug("result list has %d entries" % len(result_list))
```
Comment (shankari) [inline emission/storage/decorations/timeline.py]: You removed this line, but then we don't have an idea of how long it takes to access the cursor (e.g. to get the cursor count). Please restore a similar line in your next commit.

**Pull request merged**